adafruit_azureiot/iothub_device.py

```diff
@@ -141,7 +149,7 @@ def __init__(
         self._token_expires = token_expires
         self._logger = logger if logger is not None else logging.getLogger("log")
 
-        connection_string_values = {}
+        connection_string_values: dict = {}
```

Comment:
Thanks for pointing this out. I didn't run mpy-cross until the build failed, so I didn't realize these would fail right away.

Removed type annotation.